src/Router.jsx

```diff
@@ -1,15 +1,19 @@
-import { createBrowserRouter } from "react-router-dom";
-import App from "./App";
-import Home from "./pages/Home.jsx";
-import About from "./pages/About.jsx";
+import { createBrowserRouter } from 'react-router-dom';
+import App from './App';
+import Home from './pages/Home.jsx';
+import About from './pages/About.jsx';
+import ProductPage from './pages/ProductPage.jsx';
+import ProductListPage from './pages/ProductListPage.jsx';
 
 const router = createBrowserRouter([
   {
-    path: "/",
+    path: '/',
     element: <App />,
     children: [
-      { path: "", element: <Home /> },
-      { path: "about", element: <About /> },
+      { path: '', element: <Home /> },
+      { path: 'about', element: <About /> },
+      { path: 'products/:id', element: <ProductPage /> },
+      { path: 'products', element: <ProductListPage /> },
     ],
```

Comment:
Hardcoded paths should be saved into constant folder, I gave ortal the same comment - please talk to your team about how you want to implement it so you wont have conflicts between yourself